src/main/java/org/cba/controller/RoofTileController.java

```diff
@@ -0,0 +1,107 @@
+package org.cba.controller;
+
+import io.ebean.Ebean;
+import org.cba.domain.RoofTile;
+import org.cba.model.carport.formating.table.Row;
+import org.cba.model.carport.formating.table.TableBuilder;
+import org.cba.parameter.ParameterParser;
+import org.cba.parameter.ParameterSieve;
+import org.cba.parameter.ParsedParameters;
+import org.cba.parameter.exception.ParameterParserException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * Created by Istvan on 5/21/2017.
+ */
+public class RoofTileController extends BaseController {
```

Comment:
your controller is called RoofTile... but everywhere else you have roofTile**s**
Better rename everything to roofTile (without s) So that would be the tempate folder, in the action attribtutes in html forms and in the link in the index method